src/http.zig

```diff
@@ -742,13 +742,14 @@ pub const HTTPThread = struct {
         default_arena = Arena.init() catch unreachable;
         default_allocator = default_arena.allocator();
 
-        const loop = bun.uws.Loop.create(struct {
-            pub fn wakeup(_: *uws.Loop) callconv(.C) void {
-                http_thread.drainEvents();
-            }
-            pub fn pre(_: *uws.Loop) callconv(.C) void {}
-            pub fn post(_: *uws.Loop) callconv(.C) void {}
-        });
+        // const loop = bun.uws.Loop.create(struct {
+        //     pub fn wakeup(_: *uws.Loop) callconv(.C) void {
+        //         http_thread.drainEvents();
```

Comment:
Where is `drainEvents` called?

Reply:
Before every iteration of the event loop (`processEvents`)